src/connectors/rating-menu/__tests__/connectRatingMenu-test.js

```diff
@@ -370,116 +395,159 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/rating-menu
     });
   });
 
-  describe('routing', () => {
-    const getInitializedWidget = (config = {}) => {
-      const rendering = jest.fn();
-      const makeWidget = connectRatingMenu(rendering);
-
-      const attribute = 'grade';
-      const widget = makeWidget({
-        attribute,
-        ...config,
+  describe('getWidgetState', () => {
+    test('should give back the object unmodified if the default value is selected', () => {
+      const [widget, helper] = getInitializedWidget();
+      const uiStateBefore = {};
+      const uiStateAfter = widget.getWidgetState(uiStateBefore, {
+        searchParameters: helper.state,
+        helper,
       });
+      expect(uiStateAfter).toBe(uiStateBefore);
+    });
 
-      const initialConfig = widget.getConfiguration(new SearchParameters({}));
-      const helper = jsHelper({}, '', initialConfig);
-      helper.search = jest.fn();
+    test('should add an entry equal to the refinement', () => {
+      const [widget, helper, refine] = getInitializedWidget();
+      refine('3');
+      const uiStateBefore = {};
+      const uiStateAfter = widget.getWidgetState(uiStateBefore, {
+        searchParameters: helper.state,
+        helper,
+      });
+      expect(uiStateAfter).toMatchSnapshot();
```

Comment:
Same comment as other PRs about the snapshot, I think we can afford to change it to `toEqual` in these PRs.

Reply:
https://github.com/algolia/instantsearch.js/pull/4008/commits/46fd50601f8f5a47728166baa1aab0920eb535ba